src/OrbitGl/IntrospectionWindow.cpp

```diff
@@ -50,6 +50,21 @@ void IntrospectionWindow::StartIntrospection() {
 }
 void IntrospectionWindow::StopIntrospection() { introspection_listener_ = nullptr; }
 
+void IntrospectionWindow::Draw() {
+  ORBIT_SCOPE("IntrospectionWindow::Draw");
+  CaptureWindow::Draw();
```

Comment:
ORBIT_SCOPE_FUNCTION could be used also here

Reply:
Thanks, I missed that... changed.